### DISCUSSION TITLE
Newbie Question and backgroung information.

### DISCUSSION THREAD

**Discussion by handyMay** [Motion-Project/motion#1418] [General]
Hello, 
Thank you for building this wonderful software. I have a few background questions (mostly to understand better how this works):
1. When motion is enabled and running, does it keep the camera on all the time?
2. How to other ip motion cameras work? Do they also keep the camera on 24x7 to detect motion?
3. Is it ok for me to leave the camera running 24x7 with motion?

**Comment by tosiara** [comment]
1. camera is on all the time
2. detection is running 24/7
3. should be fine
